static/js/tools/timeline_page.tsx

```diff
@@ -15,28 +15,29 @@
  */
 
 import React, { Component } from "react";
-import { parseStatVarPath, parsePlace } from "./timeline_util";
+import { parseStatVarPath, parsePlace, getTriples } from "./timeline_util";
```

Comment:
how about call this getStatsVarProp and below too. As the object is not in triples format..

Reply:
done.